src/extensions/default/CodeFolding/Prefs.js

```diff
@@ -23,7 +22,7 @@ define(function (require, exports, module) {
                            {name: strings.SAVE_FOLD_STATES, description: strings.SAVE_FOLD_STATES_HELP});
     prefs.definePreference("alwaysUseIndentFold", "boolean", true,
                            {name: strings.ALWAYS_USE_INDENT_FOLD, description: strings.ALWAYS_USE_INDENT_FOLD_HELP});
-    prefs.definePreference("enableRegionFolding", "boolean", true,
+    prefs.definePreference("enableRegionFolding", "boolean", false,
```

Comment:
If we do not expose the region fold feature by default, should we include it in the view menu?